application/src/main/java/com/sanction/thunder/ThunderConfiguration.java

```diff
@@ -16,4 +16,22 @@
   public String getDynamoEndpoint() {
     return dynamoEndpoint;
   }
+
+  @NotNull
+  @Valid
+  @JsonProperty("application")
+  private final String application = null;
+
+  public final String getApplication() {
+    return application;
+  }
+
+  @NotNull
+  @Valid
+  @JsonProperty("secret")
+  private final String secret = null;
+
+  public final String getSecret() {
+    return secret;
+  }
```

Comment:
Maybe instead of one `String` field for the application and one for the secret, we could have a single field that is `List<Key>`?

This would allow us to add more keys super easily in the future, we just put them in the config.